packages/lego-bricks/src/components/Card/Card.module.css

```diff
@@ -4,10 +4,19 @@
   border-radius: var(--border-radius-lg);
 }
 
+.withIcon {
+  gap: var(--spacing-md);
+
+  @media (max-width: 992px) {
```

Comment:
We should definitely look more into why viewport variables in `lego-bricks` don't work